### ISSUE TITLE
weight- module

### ISSUE THREAD

**Issue by Henrilin28** [xslittlegrass/CarND-Vehicle-Detection#1]
Hi, 
I tried to reproduce you result but I found the yolo-tiny weight is not working.. Can you show me where you get the weights?

Best,
Henry

**Comment by xslittlegrass** [comment]
Hi Henry,

The weight file can be downloaded from here https://drive.google.com/file/d/0B1tW_VtY7onibmdQWE1zVERxcjQ/view?usp=sharing

Let me know if you have problems.

**Comment by Henrilin28** [comment]
HI @xslittlegrass,

It works but the same model have different result! I cant detect any car!

btw

add the following code can show the model layers

``` python
from IPython.display import SVG
from keras.utils.visualize_util import model_to_dot

SVG(model_to_dot(model).create(prog='dot', format='svg'))

```

**Comment by xslittlegrass** [comment]
Are you using the same test images in the test_images folder? Can you post your results?

**Comment by Henrilin28** [comment]
Yes, I am using the same images you provided. 

Please find my result in attachment

-- 
Chuan-Heng (Henry) Lin
Tel: +886 938-507-930 
Email: imhenry@me.com ; Website: www.henrilin28.com

在 2017年2月24日 於 上午10:36:22, xslittlegrass (notifications@github.com) 寫下:

Are you using the same test images in the test_images folder? Can you post your results?

**Comment by xslittlegrass** [comment]
You attached the files in your email and I cannot see your them, could you attach them on this issue page on Github?

**Comment by Henrilin28** [comment]
Here it is! <img width="999" alt="2017-02-24 10 37 20" src="https://cloud.githubusercontent.com/assets/5019331/23309898/7e0258fc-fa7e-11e6-99d8-79fa2a34838d.png">

**Comment by xslittlegrass** [comment]
It works fine on my system. I'm using tensorflow 0.12.0-rc0 on python 3.5.2. Could you update Keras to the newest version?

**Comment by Henrilin28** [comment]
you use tensorflow for backend?

**Comment by Henrilin28** [comment]
I use python 2.7!

**Comment by xslittlegrass** [comment]
Yes. Maybe the backend is the reason. The weights in these two backends are different, you need to convert them. There is a script in [this wiki page](https://github.com/fchollet/keras/wiki/Converting-convolution-kernels-from-Theano-to-TensorFlow-and-vice-versa) on Keras that can help you to do that.

**Comment by Henrilin28** [comment]
but you set the backend to "th", which means theano?

**Comment by xslittlegrass** [comment]
The backend and the image dimension ordering are two different settings. I set  `set_image_dim_ordering` to "th". That's only for the image dimension ordering. For instance, whether the image is organized as `[3,448,448]` ("theano" ordering) or `[448,448,3]` ("tensorflow" ordering). But I'm still using tensorflow as the backend. See [here](https://keras.io/backend/). I did this way because the pretrained weight file in yolo is organized this way.
